manager/integration/tests/test_cloning.py

```diff
@@ -387,6 +388,7 @@ def test_cloning_interrupted(client, core_api, pvc, pod, clone_pvc, clone_pod):
                                  'initiated')
 
     # Step-6
+    wait_for_volume_degraded(client, clone_volume_name)
```

Comment:
This seems not right, because we don't need to check if the cloning volume is degraded or not. The process is just:
- crash replicas of the source volume
- then check if the cloning volume is faulted and clone status is failed (here, sometimes this step is failed)

About failing to check the clone status is failed, it's a side effect of https://github.com/longhorn/longhorn-tests/commit/831ba4fb10e5b14ad1a4c723e6fff43d2932c481 which causes some crashed replicas have been recreated before finishing `crash_replica_processes`...

cc @c3y1huang

Reply:
Adding reference, https://github.com/longhorn/longhorn/issues/4045.